src/packets/packet.ts

```diff
@@ -1,84 +1,81 @@
-import { byteFormat } from "../../deps.ts";
-import { BufferReader, BufferWriter } from "../buffer.ts";
-import { WriteError } from "../constant/errors.ts";
-import { debug, log } from "../logger.ts";
+import { byteFormat } from "../../deps.ts"
+import { BufferReader, BufferWriter } from "../buffer.ts"
+import { WriteError } from "../constant/errors.ts"
+import { debug, log } from "../logger.ts"
 
-/** @ignore */
```

Comment:
Classes that work internally should not be visible to the user in the document

Reply:
I think typedoc has an exclude option to avoid this behavior.